app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+  def create
+    user = User.where(email: params[:email]).first
+    if user && user.authenticate(params[:password])
+      session[:user_id] = user.id 
+      redirect_to home_path, notice: 'You are signed in'
+    else
+      flash[:error] = 'Invalid email or password'
+      redirect_to sign_in_path
+    end
+  end
+
+  def destroy
+    session[:user_id] = nil
+    redirect_to root_path, notice: 'You are signed out.'
```

Comment:
Unless this is working already, you should also change this be something like `flash[:info]`